oem.json

```diff
@@ -4106,6 +4118,24 @@
         "description": "Secure Payment\nLets you pay with your phone. Privacy issue aside, you should probably not trust their security: https://www.bitdefender.com/blog/hotforsecurity/hackers-attack-oneplus-again-this-time-stealing-customer-details",
         "removal": "delete"
     },
+    {
+        "id": "com.oplus.qualityprotect",
+        "label": "QualityProtect",
+        "description": "Probably Oppo/OnePlus analytics",
+        "removal": "delete"
+    },
+    {
+        "id": "com.oplus.safecenter",
+        "label": "Security center",
+        "description": "Don't know what is it for. Every security function works without it",
```

Comment:
Breaks app lock feature according to this post: https://forum.xda-developers.com/t/the-oneplus-10-pro-debloat-thread.4503969/page-2#post-87920315